core/components/atoms/button/__stories__/SplitButton.story.jsx

```diff
@@ -19,7 +20,7 @@ export const splitButton = () => {
       <Button className="mr-2" aria-label="Request review">
         Request review
       </Button>
-      <div className="mb-10" style={{ width: '150px' }}>
+      <div className="mb-10 SplitButton-dropdown">
```

Comment:
Please verify and remove this styling as there is no need for this width and removing it from the story does not affect anything.

Reply:
It does affect the dropdown width which opens on click of the split button

Reply:
It does not seem to be do so, I tried on this sandbox https://codesandbox.io/s/sleepy-violet-sgteih?file=/index.js

Reply:
Also to control Dropdown width, it has props like `width`, `minWidth` and `maxWidth`.